server/src/main/java/org/opensearch/tasks/TaskCompleteResourceInfo.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.tasks;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class TaskCompleteResourceInfo {
```

Comment:
Please add java docs to all classes

Reply:
Added javadocs. Renamed this class to ```ThreadResourceInfo``` as it holds the resource information of particular execution of the thread.